### PR TITLE
Update DE Translation

### PR DESCRIPTION
**Pull request by rom-1**
Updating DE Translation, addition to https://github.com/FreshRSS/FreshRSS/pull/3453

### CODE REVIEW

File: app/i18n/de/sub.php [hunk 1]
```diff
@@ -34,10 +34,10 @@
 		),
 		'clear_cache' => 'Nicht cachen (für defekte Feeds)',
 		'content_action' => array(
-			'_' => 'Content action when fetching the article content',	// TODO - Translation
-			'append' => 'Add after existing content',	// TODO - Translation
-			'prepend' => 'Add before existing content',	// TODO - Translation
-			'replace' => 'Replace existing content',	// TODO - Translation
+			'_' => 'Behandlung von Feed-Inhalt beim Herunterladen von Artikelinhalt',
+			'append' => 'Artikelinhalt hinter Feed-Inhalt platzieren',
```
Comment (Frenzie) [inline app/i18n/de/sub.php]: Not something like einfügen?

**Review by Frenzie — APPROVED** [review]
lgtm

File: app/i18n/de/sub.php (hunk 1 above)
Comment (rom-1) [reply app/i18n/de/sub.php]: you are right, it's too early in the morning for my brain. platzieren is not the wrong word, but I guess hinzufügen simply fits best in this case.

https://github.com/FreshRSS/FreshRSS/pull/3455/commits/8a2b424b6f073632eef2cf851287ce3fbea584b3

Comment (Frenzie) [reply app/i18n/de/sub.php]: Haha, I actually thought you made a conscious decision against hinzufügen.

**Review by aledeg — APPROVED** [review]

**Pull request merged**